schema/collation_short_test_schema.json

```diff
@@ -0,0 +1,54 @@
+{"$schema": "https://json-schema.org/draft/2020-12/schema",
+ "$id": "https://github.com/unicode/conformance/collation_short_test.json'",
+ "title": "ICU Conformance test data description",
+ "description": "This documents the format of collation test data for execution",
+ "type": "object",
+ "properties": {
+   "test_type": {
+     "description": "the name collation_short",
+     "type": "string"
+   },
+   "tests": {
+     "description": "list of N test for collation each of type collation_short",
+     "type": "array",
+     "items": {
+       "type": "object",
+       "properties": {
+         "label": {
+           "description": "A numeric ID, unique for the set of tests",
+           "type": "string"
+         },
+         "s1": {
+           "description": "First string for comparison",
+           "type":" string"
+         },
+         "s2": {
+           "description": "Second string for comparison",
+           "type": "string"
+         },
+         "locale": {
+           "description": "optional field indication locale tag for running test",
+           "type": "string"
+         },
+         "compare_type": {
+           "description": "Optional method for comparing strings",
+           "type": "string"
+         },
```

Comment:
Suggestion: this should be an enumeration (oneOf)

Reply:
"enum" will be used for this and others.